README.md

```diff
@@ -712,6 +712,20 @@ Boolean to enable AcceptEnv options for specifying environment variables. Defaul
 
 - *Default*: 'USE_DEFAULTS'
 
+sshd_acceptenv_defaults
+-------------
+Array of environment variables options for AcceptEnv setting.
+
+- *Default*: [ 'LANG', 'LC_CTYPE', 'LC_NUMERIC', 'LC_TIME', 'LC_COLLATE',
+               'LC_MONETARY', 'LC_MESSAGES', 'LC_PAPER', 'LC_NAME', 'LC_ADDRESS',
+               'LC_TELEPHONE', 'LC_MEASUREMENT', 'LC_IDENTIFICATION', 'LC_ALL' ]
+
+sshd_acceptenv_custom
+-------------
+Array of custom environment variables options for AcceptEnv setting. Set here a new one if you would like to keep the default ones set in sshd_acceptenv_defaults option.
```

Comment:
I believe the comment should read something like

```
Array of custom environment variables options for AcceptEnv setting. If set, this will override the defaults specified by `sshd_acceptenv_defaults`.
```